datafusion/aggregate-functions/src/first_last.rs

```diff
@@ -393,53 +339,190 @@ impl Accumulator for FirstValueAccumulator {
     }
 }
 
-pub fn create_first_value_accumulator(
-    acc_args: AccumulatorArgs,
-) -> Result<Box<dyn Accumulator>> {
-    let mut all_sort_orders = vec![];
+/// TO BE DEPRECATED: Builtin FIRST_VALUE physical aggregate expression
```

Comment:
I don't understand this comment -- what does it mean to be deprecated? What is the alternative?

Is the idea it will be replaced with a UDF version ?

Reply:
I think it may be replaced with udaf

Reply:
Oh, It seems that we still need this